### PR TITLE
🚧 Add Products.PloneHotfix20160830

### PR DESCRIPTION
**Pull request by jone**
Install `Products.PloneHotfix20160830` for all Plone versions.
We need to test if we have problems with the patch.

/CC @4teamwork/dev-plone

### CODE REVIEW

**Comment by lukasgraf** [comment]
FYI, I deployed `Products.PloneHotfix20160830` on dev.onegovgever.ch, test.onegovgever.ch and lab.onegovgever.ch - haven't seen any issues yet, but I only did some superficial tests so far.

File: hotfixes/4.0.1.cfg [hunk 1]
```diff
@@ -25,6 +26,7 @@ Products.PloneHotfix20131210 = 1.0
 Products.PloneHotfix20150910 = 1.1
 Products.PloneHotfix20151208 = 1.0
 Products.PloneHotfix20160419 = 1.0
+Products.PloneHotfix20160830 = 1.1
```
Comment (jone) [inline hotfixes/4.0.1.cfg]: Outdated, there is already `1.3`

**Comment by jone** [comment]
> With this hotfix, we only use data from the request when the request method matches the form method. By default all forms are meant for POST requests, and in those we no longer allow prefilling data from a GET request. The same is true the other way around, we don’t fill in data from POST requests in forms that expect a GET request, but that likely does not happen often.

I think we do prefill z3cforms here and there. @lukasgraf did you notice problems regarding this patch?

File: hotfixes/4.0.1.cfg (hunk 1 above)
Comment (jone) [inline hotfixes/4.0.1.cfg]: @lukasgraf I'll update the hotfix version (for Plone versions). Do you want to re-deploy afterwards?

Comment (lukasgraf) [inline hotfixes/4.0.1.cfg]: @jone I think I should - I deployed `1.1`. I did so by using manual additions to the respective deployment buildouts though, not by extending from this branch. So go ahead and update, I'll re-deploy on my end.

**Comment by jone** [comment]
👍 updated

**Comment by lukasgraf** [comment]
@jone the removal of prefilling `POST` forms via `GET` parameters in fact breaks Task transitions in GEVER 😢 , possibly more. Please don't merge yet, this needs some more testing, I could well imagine other projects are affected by this.

**Comment by lukasgraf** [comment]
Since version `1.3` of the Hotfix, form prefilling via `GET` can be specifically allowed on a per-form basis using `Products.PloneHotfix20160830.z3c_form.ALLOW_PREFILL` - see [README](https://pypi.python.org/pypi/Products.PloneHotfix20160830).

That should allow us to whilelist all those cases where we need prefilling.

**Comment by buchi** [comment]
@jone this one should be merged now. We have newer hotfixes in #69 and while installing these we can get this one too. If GEVER has issues with this one, it can be skipped there. /cc @lukasgraf

**Comment by jone** [comment]
rebased.

**Comment by lukasgraf** [comment]
@buchi @jone I'm good with merging this one.

**Pull request merged**